taf/api/conf.py

```diff
@@ -0,0 +1,85 @@
+from shutil import copytree
+from typing import Optional
+from pathlib import Path
+from taf.api.keystore import generate_keys
+from taf.log import taf_logger
+
+
+def init(
+    path: Optional[str] = None,
+    should_generate_keys: bool = False,
```

Comment:
Now that I've had a chance to run this command a few times, I think that we don't even need this. If none of `keystore` location (location from which we want to copy the keys) and `roles_key_infos` are specified, we could ask them if they want to generate the keys. When generating new keys, the users will be prompted to enter a couple of things, so not having to press one more `y`  does not make a different. If you decide to keep it, I would name the CLI option `generate_keys`, but this parameter can still be `should_generate_keys`. Or use an alias when importing `generate_keys`